arangod/VocBase/Methods/UpgradeTasks.cpp

```diff
@@ -545,24 +544,8 @@ bool UpgradeTasks::addDefaultUserOther(TRI_vocbase_t& vocbase,
   return true;
 }
 
-bool UpgradeTasks::persistLocalDocumentIds(TRI_vocbase_t& vocbase,
-                                           arangodb::velocypack::Slice const& slice) {
-  if (EngineSelectorFeature::engineName() != MMFilesEngine::EngineName) {
-    return true;
-  }
-  Result res = basics::catchToResult([&vocbase]() -> Result {
-    MMFilesEngine* engine = static_cast<MMFilesEngine*>(EngineSelectorFeature::ENGINE);
-    return engine->persistLocalDocumentIds(vocbase);
-  });
-  return res.ok();
-}
-
 bool UpgradeTasks::renameReplicationApplierStateFiles(TRI_vocbase_t& vocbase,
                                                       arangodb::velocypack::Slice const& slice) {
-  if (EngineSelectorFeature::engineName() == MMFilesEngine::EngineName) {
```

Comment:
I'd prefer to leave at least assertion for the future

Reply:
Ok, added the assertion.